### PR TITLE
Add route to store new high scores

### PR DESCRIPTION
**Pull request by cmei4444**
Created a route `POST /scores/pacman` to update high scores table with new scores.

### CODE REVIEW

File: src/controllers/scoresController.ts [hunk 1]
```diff
@@ -35,3 +42,24 @@ async function getScoresFromDB(gameIndex: number) {
     throw err;
   }
 }
+
+function getCurrentTime(): string {
+  const today = new Date();
+  const date = `${today.getMonth()}-${today.getDate()}-${today.getFullYear()}`;
+  const time = `${today.getHours()}:${today.getMinutes()}:${today.getSeconds()}`;
+  return `${date} ${time}`;
+}
```
Comment (mukobi) [inline src/controllers/scoresController.ts]: `return new Date().toISOString().replace(/T/, ' ').replace(/\..+/, '')` gives the same result. https://stackoverflow.com/questions/10645994/how-to-format-a-utc-date-as-a-yyyy-mm-dd-hhmmss-string-using-nodejs

Comment (cmei4444) [reply src/controllers/scoresController.ts]: Done in 299ea2f

File: src/controllers/scoresController.ts [hunk 2]
```diff
@@ -35,3 +42,24 @@ async function getScoresFromDB(gameIndex: number) {
...
+
+async function postScoreToDB(gameIndex: number, score: number, name: string) {
+  try {
+    await pool.query(
+      `INSERT INTO kubercade.high_score_table (game_index, name, score, datetime)
+      VALUES ($1, $2, $3, $4)`,
+      [gameIndex, name, score, getCurrentTime()]
+    );
+  } catch (err) {
+    console.log('Query error');
```
Comment (evankanderson) [inline src/controllers/scoresController.ts]: Include `err.message` here?

File: src/controllers/scoresController.ts [hunk 3]
```diff
@@ -2,7 +2,6 @@ import express from 'express';
 import { util } from '../config/gameInfoUtil';
 import pg from 'pg';
 import { parse } from 'pg-connection-string';
-require('dotenv').config();
```
[the remark this answers is not in the document]
Comment (mukobi) [reply src/controllers/scoresController.ts]: +1. The dotenv package allows you to use .env files to set environment variables (.env is in our gitignore) which I've been using for local testing.

[the remark this answers is not in the document]
Comment (cmei4444) [reply src/controllers/scoresController.ts]: Not sure how that disappeared, thanks for catching. Fixed in 2b426e9

File: src/controllers/scoresController.ts (hunk 2 above)
Comment (cmei4444) [reply src/controllers/scoresController.ts]: Added in 2b426e9

**Pull request merged**